src/components/grid/grid.spec.js

```diff
@@ -323,62 +323,64 @@ describe("Grid", () => {
 
     describe(`when a custom padding is specified as a number in responsiveSettings,
       using one of the styled system props`, () => {
-      it("then the padding specified in a theme spacing property should have been set on the GridItem", () => {
-        const wrapper = enzymeMount(
-          <GridContainer id="testContainer">
-            <GridItem responsiveSettings={[{ p: 3, maxWidth: "1500px" }]}>
-              1
-            </GridItem>
-            <GridItem responsiveSettings={[{ pl: 3, maxWidth: "1500px" }]}>
-              2
-            </GridItem>
-            <GridItem responsiveSettings={[{ pr: 3, maxWidth: "1500px" }]}>
-              3
-            </GridItem>
-            <GridItem responsiveSettings={[{ pt: 3, maxWidth: "1500px" }]}>
-              4
-            </GridItem>
-            <GridItem responsiveSettings={[{ pb: 3, maxWidth: "1500px" }]}>
-              5
-            </GridItem>
-          </GridContainer>
-        );
+      [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11].forEach((prop) => {
```

Comment:
you could do it with spread operator and `keys` method:
`[...Array(12).keys()]`